### ISSUE TITLE
TMUX

### ISSUE THREAD

**Issue by jacobrreed** [eldritch-theme/eldritch#21]
Tmux theme

**Comment by linkarzu** [comment]
Is it possible to "contribute" to this?
I have a tmux theme, it uses catppuccin as the base, but modified it to have your colors
If I understand correctly, you create a new repo per tool, so a new repo for tmux is needed?
Let me know if this works for you or not, at least it's a starting point, I guess

![Image](https://github.com/eldritch-theme/eldritch/assets/124211755/3b6b1212-04e7-4e6b-9b96-78bf760af99f)


![Image](https://github.com/eldritch-theme/eldritch/assets/124211755/11e193ba-1652-47f2-bc47-699a3499ee60)

**Comment by jacobrreed** [comment]
> Is it possible to "contribute" to this? I have a tmux theme, it uses catppuccin as the base, but modified it to have your colors If I understand correctly, you create a new repo per tool, so a new repo for tmux is needed? Let me know if this works for you or not, at least it's a starting point, I guess
> 
> ![Image](https://private-user-images.githubusercontent.com/124211755/344428785-3b6b1212-04e7-4e6b-9b96-78bf760af99f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2OTIwMjMsIm5iZiI6MTcxOTY5MTcyMywicGF0aCI6Ii8xMjQyMTE3NTUvMzQ0NDI4Nzg1LTNiNmIxMjEyLTA0ZTctNGU2Yi05Yjk2LTc4YmY3NjBhZjk5Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyOVQyMDA4NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZDBiMDE4YmU4MGE0OGZjMjUxOWRmOTM3NjM1MWEwNDkwZWI3Nzc4NGNiMWIzZDVjN2ZiNmVhYTNlNTEzZjZmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.nE0jh_AHMHURjInv77wMwfz6Br1-zTUq7NH-qhhNojY)
> 
> ![Image](https://private-user-images.githubusercontent.com/124211755/344428803-11e193ba-1652-47f2-bc47-699a3499ee60.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2OTIwMjMsIm5iZiI6MTcxOTY5MTcyMywicGF0aCI6Ii8xMjQyMTE3NTUvMzQ0NDI4ODAzLTExZTE5M2JhLTE2NTItNDdmMi1iYzQ3LTY5OWEzNDk5ZWU2MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyOVQyMDA4NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zODk3Y2FjODU3ODQ0NjU0OTQ1ZWZiYjVlNDhlNGFmNGNiZDUwOTkzNWY5MzkwYWIwMjg4ZjNiMGJmZTU0NGNmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.zCIrEzvcbpe_yegQ6bx_Vql34f7JWzQtv39gLe-gOh4)

Id prefer not to have it based on catppuccin tbh. If we can make it similar essentially but obviously remove catppuccin branding and stuff and make it our own then yes we could contribute that

**Comment by linkarzu** [comment]
Makes sense, I don't have the time to do that right now, but if I ever do, I'll let you know!
